ui/Screen/Settings.svelte

```diff
@@ -170,11 +170,11 @@
       <div class="section-item-single">
         <div class="info">
           <p class="typo-text-bold">
-            Seeds help you find more projects and people on the network.
+            Seeds help you find more projects and users on the network.
```

Comment:
peers?

Reply:
Why not people?

Reply:
Whether they are human isn't really the point, "peers" or "users" works for me.

Reply:
I've been trying to use `users` consistently throughout the app and documentation to represent people. `Users` can will be able to manage multiple `peers`